soh/soh/Enhancements/randomizer/3drando/fill.hpp

```diff
@@ -1,16 +1,67 @@
 #pragma once
 
 #include "../randomizerTypes.h"
+#include "location_access.hpp"
+#include "../entrance.h"
 
 #include <vector>
 #include <string>
 
+//RANDOTODO merge into Logic once Logic is a class passed to logic funtions 
+struct GetAccessableLocationsStruct {
```

Comment:
I missed this when the PR with the struct got merged, but should be `GetAccessibleLocationsStruct`.